Tests/Core/Extensions/UIFont+PropertyProvidingTests.swift

```diff
@@ -10,13 +10,13 @@ import XCTest
 @testable import Frames
 
 final class UIFont_PropertyProvidingTests: XCTestCase {
-    let font = UIFont.systemFont(ofSize: 12)
+    let font = UIFont(name: "Arial", size: 12)!
 
     func test_equality() {
         XCTAssertEqual(font.properties, font.properties)
     }
 
     func test_properties() {
-        XCTAssertEqual(font.properties, [.size: 12.0, .name: ".SFUI-Regular"])
+        XCTAssertEqual(font.properties, [.size: 12.0, .name: "ArialMT"])
```

Comment:
No blockers for release, but I do wonder why we're using "Arial" for unit tests instead of SanFrancisco 🤔

Reply:
The system font in the Cocoapods generated app host (which we have no control over) was `".SFUIText"` instead of `".SFUI-Regular"` so has been set to Arial to keep it all the same